src/main/java/net/snowflake/ingest/streaming/internal/ParquetTypeGenerator.java

```diff
@@ -60,10 +60,11 @@ public void setMetadata(Map<String, String> metadata) {
    * side.
    *
    * @param column column metadata as received from server side
-   * @param id column id
+   * @param id column id if column.getOrdinal() is not available
    * @return column parquet type
    */
   static ParquetTypeInfo generateColumnParquetTypeInfo(ColumnMetadata column, int id) {
+    id = column.getOrdinal() == null ? id : column.getOrdinal();
```

Comment:
We will remove the `int id` when the server side fix is deployed, right?

Reply:
yes, I think so

Reply:
we need this for the backwards server side compatibility for the release rollback period